doc/source/user_guide/glossary.md

```diff
@@ -5,10 +5,10 @@ Work in progress
 ```{glossary}
 
 array
-    Numerical array, provided by the {class}`numpy.ndarray` object. In
+    Numerical array, provided by the :py:obj:`numpy.ndarray` object. In
```

Comment:
This a Markdown file? It looks like the rST syntax is not working here. See the [job artifact](https://output.circle-artifacts.com/output/job/30698de2-dd75-4cfd-a7a0-19e1f9f3397d/artifacts/0/doc/build/html/user_guide/glossary.html).